controllers/teams.js

```diff
@@ -1058,6 +1060,11 @@ router.get('/:teamId/members', async (req, res, next) => {
 			} else {
 				actions = addButtonTrash(actions);
 			}
+			let userRolesContainsStudent;
```

Comment:
I'd prefer initializing with false
```suggestion
			let userRolesContainsStudent = false;
```